src/HBL01MItineraryHashtag/HBL01MItineraryHashtag.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Get } from '@nestjs/common';
+
+import { HBL01MItineraryHashtagService } from './HBL01MItineraryHashtag.service';
+
+@Controller('HBL01')
+export class HBL01MItineraryHashtagController {
+  constructor(private readonly service: HBL01MItineraryHashtagService) {}
```

Comment:
https://docs.google.com/spreadsheets/d/107f2L_2REDahiB-T8hWvKK8tHgqPog-_GM_DrS0aWYs/edit#gid=542225782&range=50:50
HBL001SetMItineraryTagData のはず!!!
```suggestion
  constructor(private readonly service: HBL001SetMItineraryTagDataService) {}
```

Reply:
[5f1afdd](https://github.com/Ayato-kosaka/spelieve-backend-tag/pull/3/commits/5f1afddbef170646f4221847eb012f53689e5e24)で対応